src/components/molecules/definitionitem/DefinitionItem.jsx

```diff
@@ -4,10 +4,32 @@ import React from 'react';
 import './DefinitionItem.scss';
 
 import { SynonymTable } from '../../atomic';
+import { useStateValue } from '../../../store/store.js';
 
 const DefinitionItem = ({ payload }) => {
+	const [
```

Comment:
This should be a dumb component.  Pass these values in as props from parent instead